lib/install/management/finder.go

```diff
@@ -269,8 +269,11 @@ func (d *Dispatcher) searchVCHsPerRP(pool *object.ResourcePool) ([]*vm.VirtualMa
 	var pools []*object.ResourcePool
 	pools = append(pools, pool)
 
-	children, err := d.listChildrenPools(pool)
+	children, err := d.session.Finder.ResourcePoolList(d.ctx, path.Join(pool.InventoryPath, "..."))
```

Comment:
You can see the ResourcePoolList difference using govc too:

```
% govc-0.12.1 pool.info '*' | grep Path:
  Path:             /ha-datacenter/host/dougm-esxbox.localdomain/Resources

% govc-0.13.0 pool.info '*' | grep Path:                                                                                                                                                                                                                            
  Path:             /ha-datacenter/host/dougm-esxbox.localdomain/Resources
  Path:             /ha-datacenter/host/dougm-esxbox.localdomain/Resources/dougm-vch
```

When forcing "list" mode, they are the same:
```
% govc-0.12.1 pool.info './*' | grep Path:                                                                                                                                                                                                                          
  Path:             /ha-datacenter/host/dougm-esxbox.localdomain/Resources
% govc-0.13.0 pool.info './*' | grep Path:                                                                                                                                                                                                                          
  Path:             /ha-datacenter/host/dougm-esxbox.localdomain/Resources
```